gocd/bci.gocd.yaml

```diff
@@ -189,23 +191,25 @@ pipelines:
         - SLE
         environment_variables:
           OSC_CONFIG: /home/go/config/oscrc-totest-manager
+          BCI_TOKEN: '{{SECRET:[opensuse.secrets][BCI_TOKEN]}}'
         resources:
         - staging-bot
         tasks:
         - script: |-
+            SPRJ=SUSE:SLE-15-SP4:Update:BCI
```

Comment:
The file is autogenerated - to avoid exactly what happened here: that you forget about SP5.

Reply:
each SP has its own release pipeline, SP5 is handled in lines 281+ (old version of the script, I'll update it)

Reply:
The file is still autogenerated

Reply:
sorry, it took me a while to understand: the entire yaml file is generated from erb file !! Will adapt my PR

Reply:
That's why we have a CI in this repo - so you get email about failures :)